products/windows-nano-server.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Microsoft Nano Server
+category: os
+tags: microsoft windows
+iconSlug: windows
+permalink: /windows-nano-server
+alternate_urls:
+  - /windowsnanoserver
+versionCommand: winver
+releasePolicyLink: https://learn.microsoft.com/lifecycle/products/?terms=Windows%20Server
```

Comment:
Is there an official lifecycle link for Nano Server?

Reply:
It is a built option of Windows Server (as is Server Core). They therefore are built and maintained in parallel and not standalone. They also share therefore the lifecycle of Windows Server. But the tags and existing builds are partially separate.

https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle has some more information on the container lifecycle (which is just one aspect of those beside the installation option variant on the main product). But in my opinion, the main reason why I would love to have those pages is for the container part.

Reply:
Ok then write this into the description.

Reply:
I used this as `releasePolicyLink` on both new ones and also made all `learn.microsoft.com` links invariant (removing `en-us`) so that it will show in the browsers language.